src/deprecated.jl

```diff
@@ -391,13 +391,13 @@ end
 function aggregate(gd::GroupedDataFrame, f::Any; sort::Bool=false)
     df = combine(gd, valuecols(gd) .=> [f])
     if sort
-        Base.depwarn("`aggregate` is deprecated. Instead use" *
+        Base.depwarn("`aggregate(gd, f, sort=true)` is deprecated. Instead use" *
                      " `df = combine(gd, valuecols(gd) .=> f); " *
                      "sort!(df, setdiff(names(df), groupcols(gd)))`.",
                      :aggregate)
         sort!(df, setdiff(names(df), groupcols(gd)))
     else
-        Base.depwarn("`aggregate(gd, fs, sort=$sort)` is deprecated. Instead use" *
+        Base.depwarn("`aggregate(gd, f)` is deprecated. Instead use" *
                      " `combine(gd, [valuecols(gd) .=> f for f in fs]...)`",
```

Comment:
```suggestion
                     " `combine(gd, valuecols(gd) .=> f)`",
```

Reply:
I thought I have commented on this already - sorry for missing this.
What you propose is not correct. We have to apply all `valuecols(gd)` to all functions in `fn` (in other words - we need to do a Cartesian product). The alternative to what I have written would be e.g.:
```
(hcat(valuecols(gd)...) .=> fs)...
```
but I am not sure how many users would understand what is going on here.

Also we could write (this will be without `...`):
```
[n => f for f in fs for n in valuecols(df)]
```

So - given the options what would you prefer?

Reply:
But `fs` doesn't exist in that method?

Reply:
wow - you are right. There are so many deprecated methods. Thank you for being so precise.